demisto_sdk/commands/download/tests/downloader_test.py

```diff
@@ -203,6 +203,11 @@ def test_verify_code_lang(self, code_lang, file_type, file_name, err_msg, output
 
     @pytest.mark.parametrize('data, file_type, entity', [
         ({'name': 'test-pb'}, 'playbook', TEST_PLAYBOOKS_DIR),
+        ({'name': 'playbook_testing'}, 'playbook', PLAYBOOKS_DIR),
+        ({'name': 'playbook_test'}, 'playbook', TEST_PLAYBOOKS_DIR),
+        ({'name': 'playbookTest'}, 'playbook', TEST_PLAYBOOKS_DIR),
+        ({'name': 'Testplaybook'}, 'playbook', TEST_PLAYBOOKS_DIR),
+        ({'name': 'Test-playbook'}, 'playbook', TEST_PLAYBOOKS_DIR),
```

Comment:
Please add cases for 

`playbook_Test`
`playbook-test`
`playbook-Test`
`Test123`
`Test_123`
`test_123`
`test-123`